server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSParser.g4

```diff
@@ -449,6 +449,10 @@ cics_free_body: ((CONVID | SESSION) cics_name | STATE cics_cvda | CHILD cics_dat
 cics_freemain: (FREEMAIN | FREEMAIN64) cics_freemain_opts;
 cics_freemain_opts:(DATA cics_data_area | DATAPOINTER cics_value | cics_handle_response)+;
 
+/** GDS */
+cics_gds: GDS cics_gds_opts;
+cics_gds_opts: ~(END_EXEC|EOF)*;
```

Comment:
Let's exclude `DOT` as well so that the parser does not consume the whole document when end-exec is missing.
```suggestion
cics_gds_opts: ~(END_EXEC|EOF|DOT)*;
```